doc/like-button.md

```diff
@@ -0,0 +1,36 @@
+# like-button
+
+## Platform Support
+
+This library can be used in the **browser** only.
+
+## Installation
+
+Installing this module is fairly simple.
+
+Run `jspm install github:egeland/aurelia-facebook` from your project root.
+
+## Usage
+
+### Configuring the plugin
+
+Make sure your project uses a `main.js` file to initialize aurelia.
+In your configure function, add the following:
+
+```js
+  aurelia.use.plugin('doktordirk/aurelia-facebook')
```

Comment:
right. that wasn't planed. 

so, maybe don't merge when you still have comments :smile: